consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumerTest.java

```diff
@@ -65,46 +58,61 @@
 import uk.gov.moj.sdt.domain.api.IServiceType;
 import uk.gov.moj.sdt.domain.api.ITargetApplication;
 import uk.gov.moj.sdt.transformers.api.IConsumerTransformer;
-import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 import uk.gov.moj.sdt.ws._2013.sdt.baseschema.CreateStatusCodeType;
 import uk.gov.moj.sdt.ws._2013.sdt.baseschema.CreateStatusType;
 import uk.gov.moj.sdt.ws._2013.sdt.targetapp.indvrequestschema.HeaderType;
 import uk.gov.moj.sdt.ws._2013.sdt.targetapp.indvrequestschema.IndividualRequestType;
 import uk.gov.moj.sdt.ws._2013.sdt.targetapp.indvresponseschema.IndividualResponseType;
 import uk.gov.moj.sdt.ws._2013.sdt.targetappinternalendpoint.ITargetAppInternalEndpointPortType;
 
+import javax.xml.soap.SOAPException;
+import javax.xml.soap.SOAPFault;
+import javax.xml.ws.WebServiceException;
+import javax.xml.ws.soap.SOAPFaultException;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
 /**
  * Test class for the individual request consumer.
  *
  * @author Manoj Kulkarni
  */
-public class IndividualRequestConsumerTest extends AbstractSdtUnitTestBase {
-    /**
-     * Connection time out constant.
-     */
-    private static final long CONNECTION_TIME_OUT = 30000;
+class IndividualRequestConsumerTest {

```

Comment:
Any particular reason why the inheritance from the AbstractUnitTestBase class has been removed?

Reply:
Unused methods. Seems redundant. (Some dos to unix conversion.) Perfect for "get rid".

Reply:
To be discussed